### PR TITLE
feat(website): add experiment section

### PR DESCRIPTION
**Pull request by richbachman**
- [x] Added experiment section

### CODE REVIEW

File: packages/paste-website/src/components/homepage/Experiment.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {SITE_CONTENT_MAX_WIDTH} from '../../constants';
+
+const Experiment: React.FC = () => {
+  const theme = useTheme();
+  return (
+    <Box
+      // @ts-ignore
+      background={`linear-gradient(-180deg, ${theme.backgroundColors.colorBackgroundBody} 0%, ${theme.backgroundColors.colorBackgroundBrandHighlightLightest} 100%);`}
```
Comment (richbachman) [inline packages/paste-website/src/components/homepage/Experiment.tsx:14]: Should this be a token? Or should we allow `string` on background?

File: packages/paste-website/src/components/homepage/Experiment.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <Box textAlign="center">
+          <Heading as="h3" variant="heading10">
+            Experiment with your ideas
+          </Heading>
+          <Paragraph>Create and play with our React components in the Paste prototyping sandbox.</Paragraph>
+        </Box>
+        <Box boxShadow="shadowHigh" maxWidth="size90" marginLeft="auto" marginRight="auto">
+          <AspectRatio ratio="16:9">
+            <iframe
+              allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
```
Comment (TheSisb) [inline packages/paste-website/src/components/homepage/Experiment.tsx:28]: We can probably remove some of these safely https://developer.mozilla.org/en-US/docs/Web/HTTP/Feature_Policy/Using_Feature_Policy#The_iframe_allow_attribute

File: packages/paste-website/src/pages/index.tsx [hunk 3]
```diff
@@ -15,9 +16,10 @@ const Homepage: React.FC<{location: {pathname: string}}> = ({location}): React.R
         <meta name="description" content={SiteMetaDefaults.DESCRIPTION} />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
       </Helmet>
-      <Box paddingX="space70" maxWidth={SITE_CONTENT_MAX_WIDTH} marginX="auto">
+      <Box>
```
Comment (TheSisb) [inline packages/paste-website/src/pages/index.tsx:19]: Remove the Box too?

File: packages/paste-website/src/components/homepage/Experiment.tsx (hunk 1 above)
Comment (TheSisb) [reply packages/paste-website/src/components/homepage/Experiment.tsx:14]: This should probably be a token.

File: packages/paste-website/src/pages/index.tsx (hunk 3 above)
Comment (richbachman) [reply packages/paste-website/src/pages/index.tsx:19]: Yeah. I don't think we need it.

File: packages/paste-website/src/components/homepage/Experiment.tsx (hunk 2 above)
Comment (richbachman) [reply packages/paste-website/src/components/homepage/Experiment.tsx:28]: That's just want codesandbox provided. I doubt we really need any of them.

File: packages/paste-website/src/components/homepage/Experiment.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import {styled, themeGet} from '@twilio-paste/styling-library';
+import {AspectRatio} from '@twilio-paste/aspect-ratio';
+import {Box} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {SITE_CONTENT_MAX_WIDTH} from '../../constants';
+
+const StyledBrandGradient = styled(Box)`
```
Comment (SiTaggart) [inline packages/paste-website/src/components/homepage/Experiment.tsx:9]: Nice, a styled box is a great interim solution until we figure out how to tackle gradients. And still using the theme values as the steps 👍

File: packages/paste-website/src/components/homepage/Experiment.tsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    <StyledBrandGradient paddingX={['space90', 'space180']} paddingY="space180">
+      <Box maxWidth={SITE_CONTENT_MAX_WIDTH} marginLeft="auto" marginRight="auto">
+        <Box textAlign="center">
+          <Heading as="h3" variant="heading10">
+            Experiment with your ideas
+          </Heading>
+          <Paragraph>Create and play with our React components in the Paste prototyping sandbox.</Paragraph>
+        </Box>
+        <Box boxShadow="shadowHigh" maxWidth="size90" marginLeft="auto" marginRight="auto">
+          <AspectRatio ratio="16:9">
```
Comment (SiTaggart) [inline packages/paste-website/src/components/homepage/Experiment.tsx:28]: Whoa, an `AspectRatio` in the wild

**Review by SiTaggart — APPROVED** [review]

File: packages/paste-website/src/components/homepage/Experiment.tsx (hunk 5 above)
Comment (TheSisb) [reply packages/paste-website/src/components/homepage/Experiment.tsx:28]: 👍

**Review by TheSisb — APPROVED** [review]

**Pull request merged**